core/mailbox.h

```diff
@@ -44,6 +44,7 @@ enum MailboxType
   MUTT_MAILBOX_ANY = -2,   ///< Match any Mailbox type
   MUTT_MAILBOX_ERROR = -1, ///< Error occurred examining Mailbox
   MUTT_UNKNOWN = 0,        ///< Mailbox wasn't recognised
+  MUTT_LOCAL_PATH,         ///< Local file/directory
```

Comment:
I don't see this being used. Is it here for future use?

Reply:
Oh... It's still not used in the future (temporal grammar allowing).
There are lots of files used in the code and I wondered if the same machinery could be used to work with them.  (still undecided).  I'll drop it for now.